docs-src/concepts/what-is-a-data-converter.md

```diff
@@ -8,26 +8,22 @@ tags:
   - explanation
 ---
 
-A Data Converter is a Temporal SDK component that serializes and encodes data entering and exiting a Temporal Cluster.
-It is used by the Temporal SDK framework to serialize/deserialize data such as input and output of Activities and Workflows that need to be sent over the wire to the Temporal Cluster.
+Data Converters in Temporal are SDK components that handle the serialization and encoding of data entering and exiting a Temporal Cluster.
+Workflow input and output need to be serialized and deserialized so they can be sent as JSON to the Temporal Cluster.
 
 ![Data Converter encodes and decodes data](/diagrams/default-data-converter.svg)
 
 The Data Converter encodes data from your application to a [Payload](/concepts/what-is-a-payload) before it is sent to the Temporal Cluster in the Client call.
 When the Temporal Server sends the encoded data back to the Worker, the Data Converter decodes it for processing within your application.
-This technique ensures that all your sensitive data exists in its original format only on hosts that you control.
+This ensures that all your sensitive data exists in its original format only on hosts that you control.
 
-The main pieces of data that run through the Data Converter are arguments and return values:
+Data Converter steps are followed when data is sent to the Temporal Cluster (as input to a Workflow) and when it is returned from a Workflow (as output).
```

Comment:
I think I am *not* going to make the "Cluster" change as part of this PR since that needs to be more holistic -- but I will update the diagram label for now.